docs/csharp/language-reference/keywords/using-directive.md

```diff
@@ -2,6 +2,8 @@
 description: "using directive - C# Reference"
 title: "using directive - C# Reference"
 ms.date: 07/20/2015
+f1_keywords: 
+  - "using_CSharpKeyword"
```

Comment:
This is also on referenced in `keywords/using.md`, which would need to be changed as part of this.

Reply:
Ah, shoot, I was supposed to have already removed that.  Thanks for pointing it out and reminding me.